extensions/fairifier-rdf-extension/dtls/fairifier/RdfSkeletonService.java

```diff
@@ -13,8 +14,16 @@
  */
 
 interface RdfSkeletonService {
-    public List<RdfSkeletonTransformer> listModels(String fileType) throws IOException;
-    public List<RdfSkeletonTransformer> listModels()  throws IOException;
-    public void saveModel(String json, String fileType, String title, String proejctId)  throws IOException;
-    public String loadModel(String ProjectId)  throws IOException;
-}
\ No newline at end of file
+
+    public List<Model> listModels(String fileType)
```

Comment:
All four methods could use some method level metadata (as I've explained in Monday's presentation).